### PR TITLE
requiring user interaction before running tests, per chrome autoplay policy

### PR DESCRIPTION
**Pull request by aceslowman**
This resolves some issues I was having running the tests in Chrome

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -27,6 +27,23 @@ var allTests = [
...
 require(allTests, function(){
-  mocha.run();
+  document.getElementById('mocha').innerHTML = 'click to begin tests';
+
+  // chromes autoplay policy requires a user interaction
+  // before the audiocontext can activate
+  function mousePressed() {
+    if(!test_has_run) {
+      document.getElementById('mocha').innerHTML = '';
+      getAudioContext().resume()
```
Comment (therewasaguy) [inline test/test.js:40]: Can we use https://p5js.org/reference/#/p5/userStartAudio (in this case, `p5.prototype.userStartAudio`) for this?

Comment (aceslowman) [reply test/test.js]: This works! Just pushed new commit. When requiring p5, what is the best way to include the p5.js library? I was running into some issues before realizing that libs/p5.js was missing, which was resolved after manually importing the file, but I don't know if I saw anything about that in the contributions docs.

**Comment by endurance21** [comment]
@aceslowman would you like to merge this one!!
Go ahead

**Comment by aceslowman** [comment]
@endurance21 Great! I don't think I have permissions to merge though, or at least I'm not seeing the 'Merge Pull Request' button.

**Comment by endurance21** [comment]
Oh yes, so let me do this work🤫

**Pull request merged**